Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
The only time a user would need to apply `#if os(iOS)` in a subclass is if they plan on using that subclass for both iOS and tvOS. I realize this is a hack solution to support tvOS -- so I'm open to something more elegant -- but haven't thought of anything yet. The fact that extensions cannot have stored properties is very limiting :(